lib/foreman_theme_satellite/engine.rb

```diff
@@ -137,7 +137,13 @@ def self.documentation_version
 
   def self.documentation_root
     @documentation_root ||= begin
-      "#{documentation_server}/documentation/en-us/red_hat_satellite/#{documentation_version}/html"
+      "#{unversioned_documentation_root}/#{documentation_version}/html"
+    end
+  end
+
+  def self.unversioned_documentation_root
```

Comment:
The unversioned link will always redirect to the latest version but we already know which version the user is on. Can't we use the versioned root?

Reply:
Oh, good point. You need the target version, not the current version